app/views/volunteers/show.html.slim

```diff
@@ -12,11 +12,21 @@ nav.navbar.section-navigation
         - if @volunteer.assignments.any?
           li = button_link t_title(:new, BillingExpense), new_volunteer_billing_expense_path(@volunteer)
           li = button_link t_title(:index, BillingExpense), volunteer_billing_expenses_path(@volunteer)
+        - if @volunteer.state == Volunteer::RESIGNED
```

Comment:
If we had used [ActiveRecord::Enum](http://api.rubyonrails.org/classes/ActiveRecord/Enum.html) we could simply do `@volunteer.resigned?` :)

Unfortunately it's a bit tricky to change it now, since it uses integers instead of strings.

Reply:
The non enum decision was made by @kitsane in early stage of the project. I'm not really sure what his reasons where, so I can't really argue for or against this....

Reply:
In the very beginning of the project @kitsane strongly recommended us not to use enum, for whatever reasons he considered. I agree, it would be quite tricky to be changed now...